### PR TITLE
[FR] Cleanup `Client\Interface\Chat_console`

### PR DESCRIPTION
**Pull request by DoryanR**
Proposal for cleanup the French translation of the wiki page `Client\Interface\Chat_console`.

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
- [x] The changes have been approved on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Client/Interface/Chat_console/fr.md [hunk 1]
```diff
@@ -1,104 +1,105 @@
...
+- Cliquez sur la case `Hide Chat` ou de nouveau sur `F8` pour cacher la console de tchat.
 
-## La console de chat étendue
+## La console de tchat étendue
 
-*L'[osu!academy](/wiki/Community/Video_series/osu!academy) à déjà abordé ce sujet ici : [Episode 6 (6:52)](https://www.youtube.com/watch?v=cyYRl-a5xII) (en anglais)*
+*L'[osu!academy](/wiki/Community/Video_series/osu!academy) à déjà abordé ce sujet ici : [Episode 6 (6:52)](https://www.youtube.com/watch?v=cyYRl-a5xII) ainsi que le [multijoueur](/wiki/Gameplay/Multiplayer) (en anglais).*
 
-Sur la plupart des écrans du jeu vous pouvez cliquer sur `F9` ou sur `Online Users` pour afficher la console de chat étendue. en plus de la console de chat, le chat étendue affiche les utilisateurs connectés.
+Sur la plupart des écrans vous pouvez cliquer sur `F9` ou sur `Online Users` pour afficher la console de tchat étendue. En plus de la console de tchat, le tchat étendu superpose une liste de panneaux dans les deux tiers restants de l'écran, affichant des informations sur les utilisateurs actuellement connectés à osu!.
```
Comment (Ganondorf-1) [inline wiki/Client/Interface/Chat_console/fr.md:29]:
```suggestion
Sur la plupart des écrans, vous pouvez appuyer sur `F9` ou cliquer sur `Online Users` pour afficher la console de tchat étendue. En plus de la console de tchat, le tchat étendu superpose une liste de panneaux dans les deux tiers restants de l'écran, affichant des informations sur les utilisateurs actuellement connectés à osu!.
```

File: wiki/Client/Interface/Chat_console/fr.md [hunk 2]
```diff
@@ -1,104 +1,105 @@
...
+Chaque utilisateur connecté a un panneau qui lui est dédié. Par défaut, il affiche des informations générales (nom, score total, rang, précision, nombre de parties et photo de profil de l'utilisateur, le cas échéant). Lorsque le curseur de la souris est placé dessus, il affiche un ensemble différent d'informations (nom, rang, photo de profil le cas échéant, heure locale, fuseau horaire, localisation par pays et, si l'utilisateur l'autorise, par ville, et ce qu'il fait).
 
-- "Lock Panels" bloque l'actualisation de la liste des utilisateurs.
-- Cliquez sur un onglet pour trier les utilisateurs par nom, rang, location ou heure locale.
-- Cliquez sur "World Map" pour afficher une carte du monde avec la localisation (globale) des utilisateurs connectés.
-- Pour faire défiler la liste, vous pouvez cliquer sur le rectangle blanc et le faire descendre, ou utiliser la molette de la souris.
+- La case "Friends only" limite l'affichage à vos amis.
+- La case "Lock Panels" empêche les panneaux de se déplacer, y compris pour les nouveaux utilisateurs.
+- Cliquez sur un onglet pour trier les panels d'utilisateurs en fonction de cet attribut.
```
Comment (Ganondorf-1) [inline wiki/Client/Interface/Chat_console/fr.md:37]:
```suggestion
- Cliquez sur un onglet pour trier les panneaux d'utilisateurs en fonction de cet attribut.
```

File: wiki/Client/Interface/Chat_console/fr.md [hunk 3]
```diff
@@ -1,104 +1,105 @@
...
+| ![Le joueur modifie une beatmap](img/Chat_Console-Editing.png "Le joueur modifie une beatmap") | Rouge - Édite sa propre beatmap |
+| ![Le joueur teste une beatmap](img/Chat_Console-Testing.png "Le joueur teste une beatmap") | Violet - Test d'une beatmap, qu'il s'agisse de sa beatmap ou non. |
+| ![Le joueur soumet une beatmap](img/Chat_Console-Submitting.png "Le joueur soumet une beatmap") | Turquoise - Soumet (soit en téléchargeant, soit en mettant à jour) une beatmap qu'ils ont réalisé. |
+| ![Le joueur mod une beatmap](img/Chat_Console-Modding.png "Le joueur mod une beatmap") | Vert - Mod ou édite une beatmap de quelqu'un d'autre. |
+| ![Le joueur joue en multijoueur](img/Chat_Console-Multiplayer.png "Le joueur joue en multijoueur") | Marron - L'utilisateur est en mode multijoueur, mais ne joue pas. |
+| ![Le joueur joue en multijoueur](img/Chat_Console-Multiplaying.png "Le joueur joue en multijoueur") | Jaune - Joue actuellement en mode multijoueur. |
+| ![Le joueur est Afk](img/Chat_Console-Afk.png "Le joueur est Afk") | Noir - Inactif ou éloigné du clavier (away from keyboard) (afk). |
```
Comment (Ganondorf-1) [inline wiki/Client/Interface/Chat_console/fr.md:55]:
```suggestion
| ![Le joueur est Afk](img/Chat_Console-Afk.png "Le joueur est Afk") | Noir - Inactif (away from keyboard) (afk). |
```

File: wiki/Client/Interface/Chat_console/fr.md [hunk 4]
```diff
@@ -1,104 +1,105 @@
...
+| ![Le joueur est connecté via IRC](img/Chat_Console-IRC.png "Le joueur est connecté via IRC") | Bleu foncé sans contenu - Le joueur n'est pas dans le jeu, mais s'est connecté à un client IRC ou les statistiques ne sont pas disponibles. |
```
Comment (Ganondorf-1) [inline wiki/Client/Interface/Chat_console/fr.md:56]:
```suggestion
| ![Le joueur est connecté via IRC](img/Chat_Console-IRC.png "Le joueur est connecté via IRC") | Bleu foncé sans contenu - Le joueur n'a pas le jeu ouvert, mais s'est connecté à un client IRC ou les statistiques ne sont pas disponibles. |
```

File: wiki/Client/Interface/Chat_console/fr.md [hunk 5]
```diff
@@ -1,104 +1,105 @@
...
+
+En cliquant sur n'importe quel panneau utilisateur, un écran d'options apparaît.
+
+![Options du panneau de l'utilisateur](img/Chat_Console-Userpaneloptions.jpg "Options du panneau de l'utilisateur")
+
+1. `Observer le joueur` : Si l'utilisateur joue une beatmap et que vous l'avez, vous pouvez observer pendant qu'il joue. Votre nom figurera dans sa liste de spectateurs.
```
Comment (Ganondorf-1) [inline wiki/Client/Interface/Chat_console/fr.md:62]:
```suggestion
1. `Observer le joueur` : Si l'utilisateur joue une beatmap et que vous l'avez, vous pouvez l'observer pendant qu'il joue. Votre nom figurera dans sa liste de spectateurs.
```

File: wiki/Client/Interface/Chat_console/fr.md [hunk 6]
```diff
@@ -1,104 +1,105 @@
...
+2. `Consulter le profil` : Ouvre la page web du profil du joueur dans votre navigateur.
+3. `Discuter` : Ouvre un onglet de tchat privé avec l'utilisateur.
+4. `Inviter à rejoindre la salle` : (Si vous êtes dans une salle multijoueur) Demandez à l'utilisateur de venir dans votre salle.
```
Comment (Ganondorf-1) [inline wiki/Client/Interface/Chat_console/fr.md:65]:
```suggestion
4. `Inviter à rejoindre la salle` : (Si vous êtes dans une salle multijoueur) Demande à l'utilisateur de venir dans votre salle.
```

File: wiki/Client/Interface/Chat_console/fr.md [hunk 7]
```diff
@@ -1,104 +1,105 @@
...
+# La console de tchat
 
-# La console de chat
+Vous pouvez afficher la console de tchat d'osu! sur la plupart des écrans en appuyant sur `F8` ou sur `Show Chat` en bas à droite de l'écran.
 
-Vous pouvez afficher la console de chat sur la plupart des écrans d'osu! en appuyant sur `F8` ou sur `Show Chat` en bas à droite de l'écran.
+![La console de tchat](img/Chatconsole1.png "La console de tchat")
 
-![Chat Console](img/Chatconsole1.png "Chat Console")
+- Les onglets répertorient les salons actuellement disponibles. Il suffit de cliquer sur un onglet pour accéder à ce salon. Cliquez sur le bouton "+" jaune pour afficher une liste de nouveaux salons à rejoindre.
```
Comment (Ganondorf-1) [inline wiki/Client/Interface/Chat_console/fr.md:7]:
```suggestion
- Les onglets répertorient les canaux actuellement disponibles. Il suffit de cliquer sur un onglet pour accéder à ce canal. Cliquez sur le bouton "+" jaune pour afficher une liste de nouveaux canaux à rejoindre.
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

File: wiki/Client/Interface/Chat_console/fr.md [hunk 8]
```diff
@@ -1,104 +1,105 @@
...
+| `/j [channel]` ou `/join [channel]` | Rejoint le salon spécifié | `/join #lobby` | (L'onglet #lobby est ouvert) |
+| `/p` ou `/part` | Quitte le salon actuel. | `/part` | n/a |
+| `/unignore [user]` | Arrête d'ignorer cet utilisateur pour cette session. | `/unignore Amigo` | You may now hear Amigo. (Votre console de tchat permettra d'accéder à tout commentaire fait par Amigo.) |
+| `/me [action]` | Effectue une action à la troisième personne. | `/me est à la maison` | *John est à la maison |
+| `/msg [user] [msg]` | Envoie un message privé à `[user]`. | `/msg Amigo Je suis malade à la maison.` | (Dans l'onglet d'Amigo) John: Je suis malade à la maison. |
+| `/np` | Imprime dans le tchat la musique en cours d'écoute ou en cours de jeu. | `/np` | (Si vous jouez) * John is playing [Peter Lambert - osu! tutorial \[Gameplay Basics\]](https://osu.ppy.sh/beatmapsets/3756#osu/22538) |
+| `/reply` ou `/r` | Répondre au dernier message privé reçu. | `/r Connais tu un bon médecin ?` | (Dans l'onglet d'Amigo) \[Commentaires précédents\] John: Je suis malade à la maison. Amigo: Really? John: Connais tu un bon médecin ? |
```
Comment (DoryanR) [inline wiki/Client/Interface/Chat_console/fr.md:90]:
```suggestion
| `/reply` ou `/r` | Répondre au dernier message privé reçu. | `/r Connais tu un bon médecin ?` | (Dans l'onglet d'Amigo) \[Commentaires précédents\] John: Je suis malade à la maison. Amigo: Vraiment ? John: Connais tu un bon médecin ? |
```

File: wiki/Client/Interface/Chat_console/fr.md [hunk 9]
```diff
@@ -1,104 +1,105 @@
...
-| `/savelog` | Enregistrer une copie de la conversation actuelle dans un fichier texte. | `/savelog` | (Un dossier nommé Chaté sera crée et contiendra les sauvegardes de vos conversations)) |
-| `/watch [user]` | Commencer à regarder `[user]`. | `/watch Amigo` | Started spectating Carlito. |
-| `/nopm` | Activer/désactiver l'options « Autoriser uniquement les messages privés des amis ». | `/nopm` | (Le message apparaîtra sur votre écran) |
-| `/invite [user]` | Invites `[user]` to the multiplayer room along with the link. | `/invite Nathanael` | \* Nathanael has been invited to the game |
+| `/addfriend [user]` | Ajoute ou supprime `[user]` de votre liste d'amis. | `/addfriend Amigo` | You are now friends with Amigo. |
+| `/delfriend [user]` | Supprime `[user]` de votre liste d'amis. | `/delfriend Amigo` | You are no longer friends with Amigo. |
+| `/away [message]` | Défini un message d'absence à envoyer aux utilisateurs vous envoyant un message privé. Laissez le champ vide pour annuler. | `/away I am John Smith.` | You have been marked as being away: I am John Smith. Quand Amigo /msg John Où es-tu~ ? BanchoBot : I am John Smith. |
```
Comment (Biokax) [inline wiki/Client/Interface/Chat_console/fr.md:79]:
```suggestion
| `/away [message]` | Défini un message d'absence à envoyer aux utilisateurs vous envoyant un message privé. Laissez le champ vide pour annuler. | `/away Je suis John Smith.` | You have been marked as being away: I am John Smith. Quand Amigo /msg John Où es-tu~ ? BanchoBot : Je suis John Smith. |
```

**Review by Biokax — CHANGES_REQUESTED** [review]

**Review by Biokax — DISMISSED** [review]

File: wiki/Client/Interface/Chat_console/fr.md [hunk 10]
```diff
@@ -1,104 +1,105 @@
...
+| ![Le joueur mod une beatmap](img/Chat_Console-Modding.png "Le joueur mod une beatmap") | Vert - Mod ou édite une beatmap de quelqu'un d'autre. |
+| ![Le joueur joue en multijoueur](img/Chat_Console-Multiplayer.png "Le joueur joue en multijoueur") | Marron - L'utilisateur est en mode multijoueur, mais ne joue pas. |
+| ![Le joueur joue en multijoueur](img/Chat_Console-Multiplaying.png "Le joueur joue en multijoueur") | Jaune - Joue actuellement en mode multijoueur. |
+| ![Le joueur est Afk](img/Chat_Console-Afk.png "Le joueur est Afk") | Noir - Inactif (away from keyboard) (afk). |
+| ![Le joueur est connecté via IRC](img/Chat_Console-IRC.png "Le joueur est connecté via IRC") | Bleu foncé sans contenu - Le joueur n'a pas le jeu ouvert, mais s'est connecté à un client IRC ou les statistiques ne sont pas disponibles. |
+
+En cliquant sur n'importe quel panneau utilisateur, un écran d'options apparaît.
+
+![Options du panneau de l'utilisateur](img/Chat_Console-Userpaneloptions.jpg "Options du panneau de l'utilisateur")
```
Comment (Walavouchey) [inline wiki/Client/Interface/Chat_console/fr.md:60]: forgot one line below this image

> Press the number or click the bar to activate:

File: wiki/Client/Interface/Chat_console/fr.md [hunk 11]
```diff
@@ -1,104 +1,105 @@
...
+
```
Comment (DoryanR) [inline wiki/Client/Interface/Chat_console/fr.md:61]:
```suggestion
![Options du panneau de l'utilisateur](img/Chat_Console-Userpaneloptions.jpg "Options du panneau de l'utilisateur")

Appuyez sur le numéro ou cliquez sur la barre pour l'activer :

```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**